pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -166,47 +166,50 @@ class Build {
     def sign(VersionInfo versionInfo) {
         // Sign and archive jobs if needed
         // TODO: This version info check needs to be updated when the notarization fix gets applied to other versions.
-        if (buildConfig.TARGET_OS == "windows" || (buildConfig.TARGET_OS == "mac" && versionInfo.major == 8 && buildConfig.VARIANT != "openj9") || (buildConfig.TARGET_OS == "mac" && versionInfo.major == 13)) {
-            context.node('master') {
-                context.stage("sign") {
-                    def filter = ""
-                    def certificate = ""
-
-                    def nodeFilter = "${buildConfig.TARGET_OS}"
-
-                    if (buildConfig.TARGET_OS == "windows") {
-                        filter = "**/OpenJDK*_windows_*.zip"
-                        certificate = "C:\\Users\\jenkins\\windows.p12"
-                        nodeFilter = "${nodeFilter}&&build"
-
-                    } else if (buildConfig.TARGET_OS == "mac") {
-                        filter = "**/OpenJDK*_mac_*.tar.gz"
-                        certificate = "\"Developer ID Application: London Jamocha Community CIC\""
+        if (
+            buildConfig.TARGET_OS == "windows" ||
+        (buildConfig.TARGET_OS == "mac" && versionInfo.major == 8 && buildConfig.VARIANT != "openj9") || (buildConfig.TARGET_OS == "mac" && versionInfo.major == 13)
+        ) {
+            context.stage("sign") {
+                def filter = ""
+                def certificate = ""
+
+                def nodeFilter = "${buildConfig.TARGET_OS}"
+
+                if (buildConfig.TARGET_OS == "windows") {
+                    filter = "**/OpenJDK*_windows_*.zip"
+                    certificate = "C:\\Users\\jenkins\\windows.p12"
+                    nodeFilter = "${nodeFilter}&&build"
+
+                } else if (buildConfig.TARGET_OS == "mac") {
+                    filter = "**/OpenJDK*_mac_*.tar.gz"
+                    certificate = "\"Developer ID Application: London Jamocha Community CIC\""
+
+                    // currently only macos10.10 can sign
```

Comment:
Is that comment correct give the code below it?

Reply:
Probably not. I'll remove it